### ISSUE TITLE
remove project specific parts

### ISSUE THREAD

**Issue by matthiasbalke** [lxklssn/jiraissues#3]
Some parts in this plugin are very project specific like the JIRA issue status codes. While it is not possible to configure the status codes, I cannot use this plugin for my current project. That's why I would like to rewrite the JQL builder part. I would like to expose the JQL query which is build by the plugin to the users. This way the plugin becomes project neutral and could be used in many more projects. Also, the configuration would be simpler and more powerful as most JIRA users already know how to write complex JQL queries.

**example configuration**
```gradle

getIssues {
    filePath = "build/changelog"
    jql = "project in ('MY_PROJECT_KEY') and status in (CLOSED)"
    jiraVersions = ["Version 1.2.1", "Version 1.2.2", "Version 1.2.3"]
    jiraUsername = "my_jira_user"
    jiraPassword = "my_secret_password"
    jiraBaseUrl = "https://jira.example.com"
}
```

I'll provide a PR for this suggestion within the next week.

**Comment by lxklssn** [comment]
sounds good, looking forward to the PR

**Comment by lxklssn** [comment]
But just a thought, wouldn´t it be easier to just provide a list of version values in the configuration instead of rewriting the whole thing? You could simply provide those in the configuration.

**Comment by matthiasbalke** [comment]
Closed by merge of #4
